### PR TITLE
SPOTAUT-16164-Vm filters VNG

### PR DESCRIPTION
**Pull request by pripatra**
_This is the default pull request template. You can customize it by adding a `pull_request_template.md` at the root of your repo or inside the `.github` folder._

# Jira Ticket
https://spotinst.atlassian.net/browse/SPOTAUT-16164
https://spotinst.atlassian.net/browse/SPOTAUT-16801
_Include a link to your Jira Ticket_
_Example: [JIRAISS-1234](https://spotinst.atlassian.net/browse/JIRAISS-1234)_

# Demo

_Please add a recording of the feature/bug fix in work. if you added new routes, the recording should show the request and response for each new/changed route_

# Checklist:
- [ ] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [ ] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have validated all the requirements in the Jira task were answered
- [ ] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**)

### CODE REVIEW

File: examples/basic/main.tf [hunk 1]
```diff
@@ -42,10 +42,18 @@ module "ocean-aks-np-vng" {
...
-  vmsizes_filters_max_vcpu                  = 16
-  vmsizes_filters_min_memory_gib            = 10
-  vmsizes_filters_max_memory_gib            = 18
-  vmsizes_filters_series                    = ["D v3"]
-  vmsizes_filters_architectures             = ["X86_64"]
+  vmsizes_filters_min_vcpu                 = 4
+  vmsizes_filters_max_vcpu                 = 96
+  vmsizes_filters_min_memory_gib           = 14.0
+  vmsizes_filters_max_memory_gib           = 900.0
+  vmsizes_filters_series                   = ["NC","NV"]
```
Comment (chandra1-n) [inline examples/basic/main.tf]: keep the old value for series

File: examples/basic/main.tf [hunk 2]
```diff
@@ -42,10 +42,18 @@ module "ocean-aks-np-vng" {
...
+  vmsizes_filters_exclude_series           = ["HB","HC"]
```
Comment (chandra1-n) [inline examples/basic/main.tf]: use the values similar like series

File: examples/basic/main.tf [hunk 3]
```diff
@@ -42,10 +42,18 @@ module "ocean-aks-np-vng" {
...
+  vmsizes_filters_architectures            = ["X86_64", "INTEL64", "ARM64","AMD64"]
```
Comment (chandra1-n) [inline examples/basic/main.tf]: mention any one or two values

File: examples/basic/main.tf [hunk 4]
```diff
@@ -42,10 +42,18 @@ module "ocean-aks-np-vng" {
...
   taints                                    = [{"key":"key1","value":"value1", "effect" : "NoSchedule"}]
-  vmsizes_filters_min_vcpu                  = 2
-  vmsizes_filters_max_vcpu                  = 16
-  vmsizes_filters_min_memory_gib            = 10
-  vmsizes_filters_max_memory_gib            = 18
-  vmsizes_filters_series                    = ["D v3"]
-  vmsizes_filters_architectures             = ["X86_64"]
+  vmsizes_filters_min_vcpu                 = 4
+  vmsizes_filters_max_vcpu                 = 96
+  vmsizes_filters_min_memory_gib           = 14.0
```
Comment (chandra1-n) [inline examples/basic/main.tf]: remove .0 wherever necessary

File: examples/basic/main.tf [hunk 5]
```diff
@@ -42,10 +42,18 @@ module "ocean-aks-np-vng" {
...
+  vmsizes_filters_max_memory_gib           = 900.0
+  vmsizes_filters_series                   = ["NC","NV"]
+  vmsizes_filters_exclude_series           = ["HB","HC"]
+  vmsizes_filters_architectures            = ["X86_64", "INTEL64", "ARM64","AMD64"]
+  vmsizes_filters_accelerated_networking   = "Disabled"
+  vmsizes_filters_disk_performance         = "Standard"
+  vmsizes_filters_min_gpu                  = 4.0
+  vmsizes_filters_max_gpu                  = 8.0
+  vmsizes_filters_min_nics                 = 1
+  vmsizes_filters_vm_types                 = ["generalPurpose", "GPU"]
```
Comment (chandra1-n) [inline examples/basic/main.tf]: use other values, as these are used in cluster

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the comments

File: examples/basic/main.tf (hunk 1 above)
Comment (pripatra) [reply examples/basic/main.tf]: done

File: examples/basic/main.tf (hunk 2 above)
Comment (pripatra) [reply examples/basic/main.tf]: done

File: examples/basic/main.tf (hunk 3 above)
Comment (pripatra) [reply examples/basic/main.tf]: done

File: examples/basic/main.tf (hunk 4 above)
Comment (pripatra) [reply examples/basic/main.tf]: done

File: examples/basic/main.tf (hunk 5 above)
Comment (pripatra) [reply examples/basic/main.tf]: done

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**